internal/backoff/exponential.go

```diff
@@ -55,12 +54,45 @@ func (e exponentialOptions) validate() (err error) {
 	return err
 }
 
+func newRand() *rand.Rand {
+	return rand.New(rand.NewSource(time.Now().UnixNano()))
+}
+
 var defaultExponentialOpts = exponentialOptions{
-	base: time.Millisecond,
-	max:  time.Hour, // :shrug:
-	rand: rand.New(rand.NewSource(time.Now().UnixNano())),
+	base:    100 * time.Millisecond,
+	min:     100 * time.Millisecond,
+	max:     time.Minute,
+	newRand: newRand,
+}
+
+// DefaultExponential is an exponential backoff.Strategy configured
+// with a 100ms minimum delay, a 100ms base for a jittered exponential backoff
+// curve, and a maximum time to recovery of one minute.
+//
+// Exponential strategies are not thread safe.
+// Use the Isolate() method to obtain an independent backoff generator with the
+// same options and a referentially independent random number generator.
+var DefaultExponential = (&exponentialStrategy{
```

Comment:
If this is a `backoff.Strategy`, annotate the var with the type. Otherewise the type of `DefaultExponential` will be `func(uint) Duration`.

    var DefaultExponential Strategy = ...